app/views/claims/schools/users/index.html.erb

```diff
@@ -4,7 +4,7 @@
   <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <%= govuk_button_to(t(".add_user"), new_claims_school_user_path, method: :get) %>
+      <%= govuk_button_to(t(".add_user"), new_add_user_claims_school_users_path, method: :get) %>
```

Comment:
It might be nice to tidy this up while we're here:

```suggestion
      <%= govuk_button_link_to t(".add_user"), new_add_user_claims_school_users_path %>
```

Looks like there's [already a mix](https://github.com/DFE-Digital/itt-mentor-services/blob/d7d0e088df650806a67a30f8ff1bd99eb3f9c974/app/views/claims/support/claim_windows/index.html.erb#L12) of `govuk_button_to` vs `govuk_button_link_to` within the Claims side of the codebase, so this change wouldn't be introducing a new flavour of inconsistency.